.github/workflows/test.yml

```diff
@@ -0,0 +1,28 @@
+name: Test
+
+on: push
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
@gutakk How about added this to [cancel the previous ongoing test running](https://github.com/styfle/cancel-workflow-action) when we push a newer commit? 😄 
```
- name: Cancel previous runs
   uses: styfle/cancel-workflow-action@0.6.0
   with:
      access_token: ${{ github.token }}
```

Reply:
That was nice. Fixed in 0ee3b1889fb15e55515cd052742ff44ce133b8b5.